### PR TITLE
Create enumerable

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request we did the following: 

- Created enumerals using Ruby
- Tested Ruby Code
- Fixed Linters using Rubocop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+- definitely you can
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (algerina) [inline README.md:158]: - [x] The link to the MIT License is broken kindly fix it.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @team ,

Great job so far! However you have some minor issues to work on.

May I highlight the following: 
-  Well done working on the classes and methods, everything is working well✔
- Your code is well organized✔️



## CHANGES REQUIRED:muscle::clap:


Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: README.md (hunk 1 above)
Comment (dvcorreia1984) [reply README.md:158]: @algerina Thank you. I added the licence file.

**Review by danmainah — APPROVED** [review]
Hi @dvcorreia1984 and @MayPyone , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Implemented the change requested to ensure the MIT license link is working correctly.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**